app/src/main/res/values/strings.xml

```diff
@@ -650,7 +654,7 @@ Please report any problems you encounter via Github.</string>
     <string name="syncthing_terminated">Syncthing was terminated</string>
 
     <!-- Toast shown if syncthing failed to create a config -->
-    <string name="config_create_failed">Failed to create config file</string>
+    <string name="config_create_failed">Failed to create the config. Backup your sync data, then clear this app\'s data from Android settings.</string>
```

Comment:
"the config" ->  "configuration"

This doesn't really explain what happened, and the instructions are not clear. Why should I back up my pictures if the config creation failed? Or is this talking about keys/config? If it is, how can I even get to a screen where I can do that?

Reply:
=> Failed to create configuration. Consider backing up your sync data, then clear this app\'s data from Android settings.
It's also been there before and used by showCrashNotification if a corrupted config is about to be parsed in SyncthingService. We don't know what happened. But if that happens, the user needs to fully start over by wiping the SharedPrefs and app data. If the user has data in the "only available readwrite area com.nutomic.syncthingandroid/files", this data will be lost.